packages/compass-aggregations/src/modules/pipeline-builder/stage-editor.ts

```diff
@@ -224,8 +261,10 @@ export const loadPreviewForStagesFrom = (
   return (dispatch, getState) => {
     getState()
       .pipelineBuilder.stageEditor.stages.slice(from)
```

Comment:
we can use `pipelineFromStore` helper here as well to avoid `if` check in `forEach`

Reply:
We will have to keep it because our action dispatcher works on indexes of store but pipelineFromStore will filter out the wizards and thus the indexes will be skewed.